media-gfx/pqiv/pqiv-2.12.ebuild

```diff
@@ -46,20 +43,23 @@ pkg_setup() {
 }
 
 src_configure() {
-	local backends="gdkpixbuf"
-	use archive && backends+=",archive,archive_cbx"
-	use ffmpeg && backends+=",libav"
-	use imagemagick && backends+=",wand"
-	use pdf && backends+=",poppler"
-	use postscript && backends+=",spectre"
-	use webp && backends+=",webp"
-
-	doecho ./configure \
-		--backends-build=shared \
-		--backends=${backends} \
-		--prefix="${EPREFIX}/usr" \
-		--libdir="${EPREFIX}/usr/$(get_libdir)" \
-		--destdir="${ED}"
+	local backends=(
+		"gdkpixbuf"
+		$(usex archive "archive" "")
+		$(usex archive "archive_cbx" "")
+		$(usex ffmpeg "libav" "")
+		$(usex imagemagick "wand" "")
+		$(usex pdf "poppler" "")
+		$(usex postscript "spectre" "")
+		$(usex webp "webp" "")
+	)
+	local myconf=(
+		--backends-build=shared
+		--backends=$(printf "%s," "${backends[@]}")
+		--prefix="${EPREFIX}/usr"
+		--libdir="${EPREFIX}/usr/$(get_libdir)"
+	)
+	doecho ./configure "${myconf[@]}"
```

Comment:
Maybe use `edo` instead of `doecho`? Or maybe invoke it without a wrapper?

Reply:
Good  point, reworked into edo.